documentation/src/main/resources/_posts/2020-10-28-release-announcement-140.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Announcing Eclipse Ditto Release 1.4.0"
+published: true
+permalink: 2020-09-30-release-announcement-140.html
```

Comment:
the date in the link is wrong - must match the date of the *.md file

Reply:
Thanks for reporting!